CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 10.4
```

Comment:
The `client` and the `models` are not sync. For the model  we upgraded to `10.0` when we removed json4s and used only circe encoders and. For the `client` we upgraded to `10.0` when we removed the `useThrift` option. So `sponsor logos` which was models `9.25` has been added as a dependency in `client` `10.1`. The offset by `1` is unfortunate.